integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParserUtil.scala

```diff
@@ -124,10 +123,14 @@ object CarbonSparkSqlParserUtil {
       operationNotAllowed("Streaming is not allowed on partitioned table", partitionColumns)
     }
 
+    if (!external && fields.isEmpty) {
+      throw new MalformedCarbonCommandException("Table schema is not specified.")
```

Comment:
please change the error message "Creating table without column(s) is not supported"

Reply:
done.